### DISCUSSION TITLE
当你试图把新艾利都地铁港铁化

### DISCUSSION THREAD

**Discussion by WZ-112IceRiver** [share121/inter-knot#1497] [其他]
不得不说MTR版做出来确实很养眼
 #绝区零#设计  #MTR #新艾利都 #卧底蓝调 #手游 #港铁

![IMG_4194](https://github.com/user-attachments/assets/efc3c1ac-9824-418b-8150-fe88e823893f)
![IMG_4199](https://github.com/user-attachments/assets/d07eb301-b958-40b0-8bf2-2eda13b9f0da)
![IMG_4198](https://github.com/user-attachments/assets/56cffbeb-7108-41c7-82a3-153a8520219a)

**Comment by FurinaShark** [comment]
确实好看！

**Reply by WZ-112IceRiver** [reply]
谢谢夸奖

**Comment by ZhuYuan-148** [comment]
话说怎么感觉坐地铁人不多呢，每次头班车都只有我一个人。

**Reply by WZ-112IceRiver** [reply]
人确实不多

**Comment by lllliufeng** [comment]
非常好地铁，什么时候通十四分街

**Reply by WZ-112IceRiver** [reply]
哈哈哈哈下次一定
